docs(angular): add angular-rspack missing features and benchmarks (#30407)
nrwl/nx: docs/shared/guides/angular-rspack/introduction.md

{% /callout %}

## Known Limitations and Missing Features

The following are known limitations and missing features of Angular Rspack:

- Static Site Generation (SSG) is not supported.
- Angular's built-in support for Internationalization (i18n) is not supported.
- Server Routing is not supported - still experimental in Angular currently.
- App Engine APIs are not supported - still experimental in Angular currently.
- Optimization is not currently 1:1 with Angular's optimization - however, there are still great optimizations that are made.
  - Styles optimization for `inline-critical` and `remove-special-comments` are not yet implemented.
  - Inlining of fonts is not yet implemented.
- Web Workers are not fully supported.
- Hot Module Replacement (HMR) is partially supported.

If you have any other missing features or limitations, please [let us know](https://github.com/nrwl/angular-rspack/issues/new).

## Benchmarks

![Benchmarks](/shared/guides/angular-rspack/bundler-build-times.png)

> Below is a table of benchmarks for the different bundlers available, run against an application consisting of ~800 lazy loaded routes with ~10 components each - totaling ~8000 components.

Reviewer: Nit: sounds a little odd to read

```suggestion
Below is a table of benchmarks for different bundlers, tested on an application with ~800 lazy-loaded routes and ~10 components per route—totaling ~8000 components.
```
